src/view/adminhtml/web/js/datepicker.js

```diff
@@ -0,0 +1,22 @@
+//This script fixes incorrect positioning in modal datepickers.
+//This file should be removed after magento fixes the datepickers.
+
+require(["jquery"], function ($) {
+    $(window).on("load", function () {
+        if (window.location.pathname.includes("/admin/catalog/product/edit/id")) {
+            (($) => {
+                const show = $.datepicker._showDatepicker;
+                const datePicker = $($.datepicker.dpDiv[0]);
```

Comment:
I believe that `$.datepicker.dpDiv` is already the jQuery collection.
So you don't need to wrap it with the `$()` again.

Reply:
It returns me just the element itself without this some methods didn't work

Reply:
Sorry, but I don't believe you. The `$.datepicker.dpDiv` definitely is the jQuery object and has all necessary methods you need. So please use it instead of creating a new one.
![image](https://user-images.githubusercontent.com/73945186/175366451-76fdda7f-1854-48a8-8073-317f561c82e2.png)